packages/tabs/src/Tabs.ts

```diff
@@ -235,12 +235,35 @@ export class Tabs extends SizedMixin(Focusable, { noDefaultSize: true }) {
         return this.rovingTabindexController.focusInElement || this;
     }
 
+    private limitDeltaToInterval(min: number, max: number) {
```

Comment:
nit: this can be a one liner

`Math.max(min, Math.min(max, delta));`

Reply:
I did it this way previously but I think this approach is better for readability purposes.